### PR TITLE
add param type and forcenew in data source docs

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**

### CODE REVIEW

File: docs/data-sources/dcs_az.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ data "huaweicloud_dcs_az" "az1" {
...
-* `region` - (Optional) The region in which to obtain the dcs az. If omitted, the provider-level region will be used.
+* `region` - (Optional, String) The region in which to obtain the dcs az. If omitted, the provider-level region will be used.
 
-* `name` - (Optional) Indicates the name of an AZ.
+* `name` - (Optional, String) Indicates the name of an AZ.
 
-* `code` - (Optional) Indicates the code of an AZ.
+* `code` - (Optional, String) Indicates the code of an AZ.
 
 * `port` - (Optional) Indicates the port number of an AZ.
```
Comment (niuzhenguo) [inline docs/data-sources/dcs_az.md]: Missing type for `port`

File: docs/data-sources/cce_cluster.md [hunk 2]
```diff
@@ -22,15 +22,15 @@ data "huaweicloud_cce_cluster" "cluster" {
...
 The following arguments are supported:
 
-* `region` - (Optional) The region in which to obtain the cce clusters. If omitted, the provider-level region will be used.
+* `region` - (Optional, String) The region in which to obtain the cce clusters. If omitted, the provider-level region will be used.
 
-* `name` -  (Optional)The Name of the cluster resource.
+* `name` -  (Optional, String)The Name of the cluster resource.
  
-* `id` - (Optional) The ID of container cluster.
+* `id` - (Optional, String, String) The ID of container cluster.
```
Comment (niuzhenguo) [inline docs/data-sources/cce_cluster.md]: duplicate `String`

File: docs/data-sources/vbs_backup_policy.md [hunk 3]
```diff
@@ -27,19 +27,19 @@ data "huaweicloud_vbs_backup_policy" "policies" {
...
+* `id` - (Optional, String) The ID of the specific VBS backup policy to retrieve.
 
-* `name` (Optional) - The name of the specific VBS backup policy to retrieve.
+* `name` - (Optional, String) The name of the specific VBS backup policy to retrieve.
 
-* `status` (Optional) - The status of the specific VBS backup policy to retrieve. The values can be ON or OFF
+* `status` - (Optional, String) The status of the specific VBS backup policy to retrieve. The values can be ON or OFF
 
-**filter_tags** **- (Optional)** Represents the list of tags. Backup policy with these tags will be filtered.
+**filter_tags** **- (Optional, List)** Represents the list of tags. Backup policy with these tags will be filtered.
```
Comment (niuzhenguo) [inline docs/data-sources/vbs_backup_policy.md]: update this to * `filter_tags` - () like other arguments

File: docs/data-sources/cce_cluster.md (hunk 2 above)
Comment (Jason-Zhang9309) [reply docs/data-sources/cce_cluster.md]: fixed

File: docs/data-sources/dcs_az.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/data-sources/dcs_az.md]: fixed

File: docs/data-sources/vbs_backup_policy.md (hunk 3 above)
Comment (Jason-Zhang9309) [reply docs/data-sources/vbs_backup_policy.md]: fixed

**Pull request merged**